From 26c6f5cd2000c20250e8753086ab6b4a770b6ecd Mon Sep 17 00:00:00 2001
From: sleepy_chord <sj702d5@foxmail.com>
Date: Sun, 27 Sep 2015 11:03:47 +0800
Subject: [PATCH 006/150] ignore the info of rubymine

---
 .gitignore | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.gitignore b/.gitignore
index 050c9d9..03a2700 100644
--- a/.gitignore
+++ b/.gitignore
@@ -15,3 +15,6 @@
 /log/*
 !/log/.keep
 /tmp
+
+# Ignore ide(rubymine) info
+/.idea/*
-- 
1.9.1

