From d8e48b5007b19e370f5d1ff22e677db073df3890 Mon Sep 17 00:00:00 2001
From: Alex Wang <ice_b0und@hotmail.com>
Date: Thu, 1 Oct 2015 20:59:35 +0800
Subject: [PATCH 014/150] merge dev

---
 .gitignore | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index ae0c577..a69df3a 100644
--- a/.gitignore
+++ b/.gitignore
@@ -16,4 +16,5 @@
 !/log/.keep
 /tmp
 
-/.idea
\ No newline at end of file
+/.idea
+/*~
\ No newline at end of file
-- 
1.9.1

