From 10c04465d2f1a3c0647965f893ee4351d9119e88 Mon Sep 17 00:00:00 2001
From: sleepy_chord <sj702d5@foxmail.com>
Date: Fri, 2 Oct 2015 18:45:21 +0800
Subject: [PATCH 022/150] ~

---
 .gitignore | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index bac6ec5..f853578 100644
--- a/.gitignore
+++ b/.gitignore
@@ -18,5 +18,5 @@
 # Ignore ide(rubymine) info
 /.idea
 
-/*~
+*~
 
-- 
1.9.1

