From 1d177762c5a826d96c27c4aa23823debe8fe9626 Mon Sep 17 00:00:00 2001
From: Alex Wang <ice_b0und@hotmail.com>
Date: Wed, 7 Oct 2015 14:48:45 +0800
Subject: [PATCH 031/150] fix participation bugs

---
 app/models/participation.rb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/app/models/participation.rb b/app/models/participation.rb
index 2eaa0f6..4e20d70 100644
--- a/app/models/participation.rb
+++ b/app/models/participation.rb
@@ -1,8 +1,8 @@
 class Participation < ActiveRecord::Base
 
   validates :role, presence: true
-  validates_associated :users
-  validates_associated :courses
+  validates_associated :user
+  validates_associated :course
 
   belongs_to :user
   belongs_to :course
-- 
1.9.1

