From 1a079aadb424fc12a7657c689bf5bcf4eba2af14 Mon Sep 17 00:00:00 2001
From: Alex Wang <ice_b0und@hotmail.com>
Date: Thu, 8 Oct 2015 19:37:32 +0800
Subject: [PATCH 037/150] make up data for app

---
 db/init_data/db.rb | 20 +++++++++++++++-----
 1 file changed, 15 insertions(+), 5 deletions(-)

diff --git a/db/init_data/db.rb b/db/init_data/db.rb
index cb55414..b227638 100644
--- a/db/init_data/db.rb
+++ b/db/init_data/db.rb
@@ -1,6 +1,16 @@
-c = Course.create(name: 'os', description: '123')
-u = User.create(name:'alex', phone:'123', email:'a@b.c', password:'123', password_confirmation:'123')
-p = Participation.create(user_id: u.id, course_id: c.id)
-u.participations << p
-u.save
+c1 = Course.create(name: 'os', description: '123')
+c2 = Course.create(name: 'ds', description: '中文')
+c3 = Course.create(name: '语文', description: 'fuck')
+
+u1 = User.create(name:'alex', phone:'123', email:'a@b.c', password:'123', password_confirmation:'123')
+u2 = User.create(name:'betty', phone:'222', email:'b@c.d', password:'123', password_confirmation:'123')
+u3 = User.create(name:'ciara', phone:'333', email:'c@d.e', password:'123', password_confirmation:'123')
+
+
+p11 = Participation.create(user_id: u1.id, course_id: c1.id, role: ROLE_STUDENT)
+p12 = Participation.create(user_id: u1.id, course_id: c2.id, role: ROLE_ASSISTANT)
+p21 = Participation.create(user_id: u2.id, course_id: c1.id, role: ROLE_STUDENT)
+p22 = Participation.create(user_id: u2.id, course_id: c2.id, role: ROLE_STUDENT)
+p32 = Participation.create(user_id: u3.id, course_id: c3.id, role: ROLE_STUDENT)
+p33 = Participation.create(user_id: u3.id, course_id: c3.id, role: ROLE_STUDENT)
 
-- 
1.9.1

