From 2db2e0b963de75a8a49bf299d7282d2852393011 Mon Sep 17 00:00:00 2001
From: Alex Wang <ice_b0und@hotmail.com>
Date: Fri, 9 Oct 2015 14:46:50 +0800
Subject: [PATCH 043/150] fix assistants/students bugs

---
 app/controllers/courses_controller.rb | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/app/controllers/courses_controller.rb b/app/controllers/courses_controller.rb
index 29a5461..f519e00 100644
--- a/app/controllers/courses_controller.rb
+++ b/app/controllers/courses_controller.rb
@@ -4,9 +4,11 @@ class CoursesController < ApplicationController
   before_action only: [:index] do
     if params[:student_id]
       @id_key = :student_id
+      @role = ROLE_STUDENT
       check_token(params[:token], params[@id_key])
     elsif params[:assistant_id]
       @id_key = :assistant_id
+      @role = ROLE_ASSISTANT
       check_token(params[:token], params[@id_key])
     end
   end
@@ -17,7 +19,7 @@ class CoursesController < ApplicationController
     if @id_key
       @courses = Course.none
       User.find(params[@id_key]).participations.each do |participation|
-        @courses <<= participation.course
+        @courses <<= participation.course if @role == participation.role
       end
     else
       @courses = Course.all
-- 
1.9.1

