From 0cf2daf77d3bd01b866b1b9dacf572228e74efcd Mon Sep 17 00:00:00 2001
From: sleepy_chord <sj702d5@foxmail.com>
Date: Fri, 9 Oct 2015 17:17:24 +0800
Subject: [PATCH 048/150] main page

---
 config/routes.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/routes.rb b/config/routes.rb
index bdbde9f..73f6826 100644
--- a/config/routes.rb
+++ b/config/routes.rb
@@ -14,7 +14,7 @@ Rails.application.routes.draw do
   # Example of regular route:
   #   get 'products/:id' => 'catalog#view'
   get 'test' => 'test#index'
-  get 'manage' => 'manage#index'
+  get 'manage' => 'manage#main_page'
   get 'manage/MainPage' => 'manage#main_page'
   get 'manage/CourseScore/:id' => 'manage#course_score'
   get 'manage/ClassScore/:id' => 'manage#class_score'
-- 
1.9.1

