From 00182ce37205a52d3532d6c372fc6e8b8bae37ea Mon Sep 17 00:00:00 2001
From: sleepy_chord <sj702d5@foxmail.com>
Date: Sat, 17 Oct 2015 15:20:56 +0800
Subject: [PATCH 095/150] PAPA-53 ignore uploads

---
 .gitignore | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index 0a61739..02fc04c 100644
--- a/.gitignore
+++ b/.gitignore
@@ -21,4 +21,4 @@
 *~
 # ignore mp4
 /public/*.mp4
-
+/public/uploads/
-- 
1.9.1

