From d2b8a0ddbe1089c26e6b1a6a7bec19669160006d Mon Sep 17 00:00:00 2001
From: Sleepy_chord <sj702d5@foxmail.com>
Date: Fri, 23 Oct 2015 13:05:38 +0800
Subject: [PATCH 115/150] PAPA-53 add some apis frontly to add simple data

---
 app/views/test/index.html.erb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/app/views/test/index.html.erb b/app/views/test/index.html.erb
index 1dbf040..a2ec87c 100644
--- a/app/views/test/index.html.erb
+++ b/app/views/test/index.html.erb
@@ -11,3 +11,7 @@
   <button type="submit">add teacher</button>
 </form>
 
+<form action="/courses/1/students/1.json" method="post">
+  <button type="submit">add students</button>
+</form>
+
-- 
1.9.1

