From 1be845373b7297d8c5155ad40c63e48682745141 Mon Sep 17 00:00:00 2001
From: Alex Wang <ice_b0und@hotmail.com>
Date: Sat, 24 Oct 2015 12:28:35 +0800
Subject: [PATCH 134/150] 1

---
 Gemfile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Gemfile b/Gemfile
index 3931257..a46ef6b 100644
--- a/Gemfile
+++ b/Gemfile
@@ -1,5 +1,6 @@
 source 'https://ruby.taobao.org'
 
+###
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.4'
-- 
1.9.1

